### DISCUSSION TITLE
Using Vercel CLI after moving project to a new team

### DISCUSSION THREAD

**Discussion by jocruz** [vercel/community#3066] [Help] [answered]
### Summary

I decided to try out the vercel pro trial and I created a new team. I then moved my existing project from hobby to the team that uses pro. When I use the command 

vercel --prod

It asks me to ? Set up and deploy  my project

I logged in again, I used vercel switch to switch to the correct team, however when I use vercel --prod it no longer points to the existing project on the dashboard.

If I run Vercel Link command I still get ask to set up and deploy my project even though it already exists on the dashboard I do not want to make a duplicate.

### Steps to Reproduce

Moving a project from Hobby to a Pro team and trying to run the command vercel -- prod after it has been moved.

**Comment by amyegan** [comment, marked as answer]
When connecting the project via CLI, you should have the option to connect your local project with the existing project on your team. For me, it looks like this:

```bash
❯ vercel link
Vercel CLI 31.0.1
? Set up and deploy “~/discussions/cron-variety-test”? [Y/n] y
? Which scope do you want to deploy to? Amy Egan
? Found project “amyegan/cron-variety-test”. Link to it? [Y/n] y
```
